packages/reakit/src/Button/__examples__/ButtonWithTootltip/__tests__/index-test.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from "react";
+import { click, press, render } from "reakit-test-utils";
+import ButtonWithTooltip from "..";
+
+test("two-state button with tooltip", () => {
+  const { getByText: text } = render(<ButtonWithTooltip />);
+  const buttonUnlocked = text("🔓");
+  expect(buttonUnlocked).toBeVisible();
+  press.Tab();
+  expect(text("Click to lock")).toBeVisible();
+  expect(text("It's unlocked!")).toBeVisible();
+  click(buttonUnlocked);
+  const buttonLocked = text("🔒");
+  expect(buttonLocked).toBeVisible();
+  expect(text("Click to unlock")).toBeVisible();
+  expect(text("It's locked!")).toBeVisible();
+  press.Space(buttonLocked);
```

Comment:
If it already has focus, you don't need to pass the element.
```suggestion
  press.Space();
```